core/provisioning-java/src/main/java/org/apache/syncope/core/provisioning/java/data/UserDataBinderImpl.java

```diff
@@ -414,8 +415,10 @@ private boolean isPasswordMapped(final ExternalResource resource) {
                     ResourceOperation.UPDATE,
                     anyUtils.getAllResources(toBeUpdated).stream().
                             filter(resource -> resource.getProvision(toBeUpdated.getType()).isPresent()).
-                            filter(resource -> mappingManager.hasMustChangePassword(
-                            resource.getProvision(toBeUpdated.getType()).get())).
+                            filter(resource -> {
```

Comment:
Why isn't the first `filter(resource -> resource.getProvision(toBeUpdated.getType()).isPresent())` enough?

Reply:
Good point. I missed the filter actually and will revert this.